source/unpromise.d.ts

```diff
@@ -0,0 +1,10 @@
+/**
+Unwraps `Promise` and returns lifted type.
```

Comment:
This description could be clearer and more verbose on what it does.

Reply:
```ts
/**
 * Returns the type that is wrapped inside a `Promise` type.
 * 
 * @example
 * import { asyncFunction } from 'api';
 * import { UnwrapPromise } from 'type-fest';
 * 
 * let data: UnwrapPromise<ReturnType<typeof asyncFunction>>;
 * 
 * async function setValue () {
 *   data = await asyncFunction();
 * }
 * 
 * setValue();
 */
```
How's this, bud?

Reply:
👍

UnwrapPromise => Unpromise